core/client/hdfs/src/main/java/alluxio/hadoop/AbstractFileSystem.java

```diff
@@ -420,6 +420,7 @@ public void setPermission(Path path, FsPermission permission) throws IOException
    *      org.apache.hadoop.conf.Configuration)
    */
   //@Override This doesn't exist in Hadoop 1.x, so cannot put {@literal @Override}.
+  @Override
```

Comment:
see comment on above line

Reply:
this slipped through and broke the hadoop-1 build when I tried to generate RC3 artifacts